beacon_node/lighthouse_network/src/service/mod.rs

```diff
@@ -912,13 +903,11 @@ impl<E: EthSpec> Network<E> {
             }
         }
 
-        if let Err(e) = self.gossipsub_mut().report_message_validation_result(
+        self.gossipsub_mut().report_message_validation_result(
             &message_id,
             propagation_source,
             validation_result,
-        ) {
-            warn!(self.log, "Failed to report message validation"; "message_id" => %message_id, "peer_id" => %propagation_source, "error" => ?e);
-        }
```

Comment:
And these ones?

Reply:
same reason, we no longer fail at this level, message validation report always succeeds